content/4_Gold/Digit_DP.mdx

```diff
@@ -0,0 +1,152 @@
+---
+id: digit-dp
+title: 'Digit DP'
+author: Jesse Choe
+prerequisites:
+  - intro-dp
+description: 'Finding the number of integers in a range that have a property.'
+frequency: 1
+---
+
+<FocusProblem problem='focus' />
+
+## General Resources
+
+<Resources>
+	<Resource
+		source='AR'
+		title='Dynamic Programming for Computing Contests'
+		url='https://dp-book.com/Dynamic_Programming.pdf'
+	>
+
+	</Resource>
+	<Resource
+		source="GFG"
+		title="Digit DP | Introduction"
+		url="https://www.geeksforgeeks.org/digit-dp-introduction/"
+	>
+
+	</Resource>
+	<Resource
+		source="Youtube"
+		title="Introduction to Digit Dynamic Programming"
+		url="https://www.youtube.com/watch?v=heUFId6Qd1A"
+	>
+		Great introduction video
+	</Resource>
+	<Resource
+		source="CF"
+		title="Digit DP"
+		url="https://codeforces.com/blog/entry/53960"
+	>
+
+	</Resource>
+</Resources>
+
+## Solution - Odometer
```

Comment:
seems like a sus idea to jump right into an example without a general overview of what this is